tests/behavioural/features/steps/common.py

```diff
@@ -44,6 +46,12 @@ def step_impl_use_mock_case_service(context):
     case_service.use_mock_service()
 
 
+@given("using mock internal user service")
+def step_impl_use_mock_internal_user_service(context):
+    """ use the default party service tests"""
```

Comment:
🎨 Minor cosmetic - consistency - should doc strings start with upper or lower case chars? Seems to have a mix here.

Reply:
Should be done right 👍